content/60.methods.md

```diff
@@ -54,87 +105,35 @@ For instance, the template variable {% raw %}`{{ebm_trials_results}}`{% endraw %
 The template variables also include versioned URLs to the dynamically updated figures.
 The JSON files and figures are stored in the `external-resources` branch of the manuscript's GitHub repository, which acts as versioned storage.
 The GitHub Actions workflow automatically adds and commits the new JSON files and figures to the `external-resources` branch every time it runs, and Manubot uses the latest version of these resources when it builds the manuscript.
-
 The workflow file is available from <https://github.com/greenelab/covid19-review/blob/master/.github/workflows/update-external-resources.yaml> and the scripts are available from <https://github.com/greenelab/covid19-review/tree/external-resources>.
 The Python package versions are available in <https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml>.
-<!-- These files are archived with [Software Heritage](...). -->
+<!-- To Do: These files are archived with [Software Heritage](...). -->
```

Comment:
Software Heritage failed to archive this repository when I tried last year.  We may need to use Zenodo or something else.

Reply:
This was in the draft manuscript so I just moved it here! But definitely happy to change to whatever works!